src/Xablu.WebApiClient/WebApiClient.cs

```diff
@@ -22,12 +22,10 @@ public class WebApiClient : IWebApiClient
         private Lazy<HttpClient> _background;
         private Lazy<HttpClient> _userInitiated;
         private Lazy<HttpClient> _speculative;
-        private Func<HttpMessageHandler> httpHandler;
+        private Func<HttpMessageHandler> httpHandler = () => new HttpClientHandler();
```

Comment:
Yeah I noticed, that's why I mentioned its just nit picking. But good you fixed it (all about being a good boyscout ;) )